src/dashboard/components/options/OptionsHome.js

```diff
@@ -43,42 +72,35 @@ export default function OptionsHome(props) {
           <h4>{t('Language')}</h4>
           <LanguagePicker />
         </div>
-        <hr />
-        <h1 className="secondary-text">{t('Coming Soon!')}</h1>
-        <div className="dashboard__options__item disabled">
-          <h4>{t('Auto-save')}</h4>
-          <Switch
-            disabled
-            isOn={!!settings.user.autoSave || true}
-            handleToggle={() => saveSetting('user.autoSave', !settings.user.autoSave)}
-            labelText={t('By default, use auto-save for projects')}
-          />
-        </div>
-        <div className="dashboard__options__item disabled">
-          <h4>{t('Days of Backup')}</h4>
+        <div className="dashboard__options__item">
+          <h4>{t('Backup Storage Type')}</h4>
           <FormGroup controlId="backupDays">
-            <ControlLabel>{t('Number of Days of Rolling Backups to Keep')}</ControlLabel>
-            <FormControl
-              disabled
-              type="number"
-              value={settings.user.backupDays || 30}
-              onChange={(event) => saveSetting('user.backupDays', Number(event.target.value))}
-            />
-            <HelpBlock>{t('Backups older than this will be erased')}</HelpBlock>
+            {renderDropdown()}
```

Comment:
Looks good to me. As for my coding style, i would render it directly here so i would not need to create a function.
I prefer to separate them on a function if it has too much condition and it gets complicated like having a switch statement